### PR TITLE
PLF-7255 Share Activities with Multiple documents

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA, or see the FSF
+ * site: http://www.fsf.org.
+ */
+
+(function(ecm_bootstrap, gj, base, wcm_utils) {
```
Comment (thomasdelhomenie) [inline extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js]: Don't we already have a multiupload component (used in File Explorer) ?

File: extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js [hunk 2]
```diff
@@ -0,0 +1,576 @@
...
+     *          uploadIds identifier upload
+     */
+    initUploadEntry : function(id, uploadIds, limitCount, limitSizeInMB, megabyteMessage) {
+      uiUploadInput.megabyteMessage = megabyteMessage;
+
+      var obj = $("#"+id).find("#DropFileBox");
+      obj.on('dragenter', function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        $(this).css('background', '#ccc');
```
Comment (thomasdelhomenie) [inline extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js]: We should use css class instead of hardcoding value.

Comment (boubaker) [inline extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js]: sounds good

File: extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js (hunk 1 above)
Comment (boubaker) [inline extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js]: Not adequate for Composer Use case.

File: extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js (hunk 2 above)
Comment (boubaker) [inline extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js]: done

File: extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js (hunk 1 above)
Comment (thomasdelhomenie) [inline extension/resources/src/main/webapp/javascript/eXo/social/webui/ComposerMultiUpload.js]: ok, that's too bad. I think we should have a more generic multi-upload component which could be use in both features, but not in the scope of this feature.

**Pull request merged**